File: backend/Multibanca.Domain.Models/Multibanca/registrar_fecha_registro_cbr.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class registrar_fecha_registro_cbr : base_auditoria
    {
        public int id_registrar_fecha_registro_cbr { get; set; }
        public long id_expediente { get; set; }
        public DateTime? fecha_registro_cbr { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/registrar_fecha_registro_cbr/models/registrar_fecha_registro_cbr.ts
export interface RegistrarFechaRegistroCbr {
  id_registrar_fecha_registro_cbr: number;
  id_expediente: number;
  fecha_registro_cbr: string | null;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}
